### DISCUSSION TITLE
[FEATURE] install via apt (package manager)

### DISCUSSION THREAD

**Discussion by js1972** [awslabs/amazon-kinesis-video-streams-producer-sdk-cpp#963] [Ideas]
**Is your feature request related to a problem? Please describe.**
Its clunky having to manually set env vars and building from source. Why cant we install via apt with the kvssink gstreamer plugin installed into the correct location.

**Describe the solution you'd like**
I'd like to be able to do :
```
sudo apt install kvssink (or kvs producer etc)
gst-inspect-1.0 kvssink
```

**Comment by MushMal** [comment]
This is a valid request. Ideally, kvssink should be part of the GStreamer plugin package but we haven’t got around yet it pushing it

**Comment by js1972** [comment]
Eagerly looking forward to a proper gstreamer package. With the current build progress I'm unable to build in a docker container as well - get openssl conflicts with boto3.

**Comment by ishangupta3** [comment]
@js1972 it is me from the nvidia forum; I agree and looking forward to a proper gstreamer package.

**Comment by hassanctech** [comment]
We're discussing this internally and hope to work on this sometime in the near future, but cannot provide an ETA at the moment.

**Comment by rkacenski** [comment]
Any progress on this? I'm currently trying to build the plugin with dpkg-buildpackage will little success
